common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_common.yaml

```diff
@@ -660,6 +660,23 @@ sDeviceInfo:
   wan_ip_address: uint32_t
   wan_network_mask: uint32_t
 
+sIfaceInfo:
+  _type: struct
+  iface_name:
+    _type: char 
+    _length: [ "beerocks::message::IFACE_NAME_LENGTH" ]
+  iface_mac: sMacAddr
+  iface_status: beerocks::eNodeState
+
+sDeviceData:
+  _type: struct
+  al_mac: sMacAddr
+  radios:
+    _type: sIfaceInfo
+    _length: [ "beerocks::message::DEV_MAX_RADIOS" ] 
```

Comment:
You should consider declaring this as tlvf class where variable length list is supported. Then there is no need for hard coding max radios.
So:
```
cDeviceData:
  _type: class
  al_mac: sMacAddr
  num_radios: uint8_t
  radios:
    _type: sIface_info
    _length: [ num_radios ]
    _length_var: True

Reply:
I considered creating beerocks_message_common_classes.yaml and adding it there.
After receiving the message i would still need to use a struct with const array to copy the message (BML_DEVICE_DATA) so not sure there is added value doing it.
Also i'm using sDeviceData in other places.

Prefer to leave it a const array.